src/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { InternalLink } from "./Link";
+import { Button as MaterialButton } from "@material-ui/core";
+import { ButtonProps } from "@material-ui/core/Button/Button";
+
+interface Props {
+  /** Supports both internal and external links */
+  to?: string;
+  component?: any;
```

Comment:
s/any/React.Component/

Reply:
component?: HTMLElement;

Reply:
@josephsintum HTMLElement is a default HTML element not a React component - you should use `React.ComponentType<any>`